src/test/java/JdiHomework/main/site/sections/ResultsSection.java

```diff
@@ -0,0 +1,45 @@
+package JdiHomework.main.site.sections;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import JdiHomework.main.entities.MetalsAndColorsDataset;
+
+public class ResultsSection extends Section {
+
+    @UI(".summ-res")
+    Text summary;
+    @UI(".col-res")
+    Text color;
+    @UI(".met-res")
+    Text metal;
+    @UI(".sal-res")
+    Text vegetables;
+    @UI(".elem-res")
+    Text elements;
+
+    @Override
+    public String toString(){
+        return String.format("Sum: %s\n color: %s\nmetal: %s\nvegetables: %s\n elements: %s\n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.